qiskit_ibm_provider/transpiler/passes/scheduling/dynamical_decoupling.py

```diff
@@ -100,29 +100,35 @@ def uhrig_pulse_location(k):
     .. note::
 
         You need to call
-        :class:`~qiskit_ibm_provider.transpiler.passes.scheduling.DynamicCircuitScheduleAnalysis`
+        :class:`~qiskit_ibm_provider.transpiler.passes.scheduling.ALAPScheduleAnalysis`
         before running dynamical decoupling to guarantee your circuit satisfies acquisition
         alignment constraints for dynamic circuit backends.
     """
 
     def __init__(
         self,
         durations: InstructionDurations,
-        dd_sequence: List[Gate],
+        dd_sequences: Union[List[Gate], List[List[Gate]]],
         qubits: Optional[List[int]] = None,
-        spacing: Optional[List[float]] = None,
+        spacings: Optional[Union[List[List[float]], List[float]]] = None,
         skip_reset_qubits: bool = True,
         pulse_alignment: int = 1,
         extra_slack_distribution: str = "middle",
+        sequence_min_length_ratios: Optional[Union[int, List[int]]] = None,
+        insert_multiple_cycles: bool = False,
     ):
         """Dynamical decoupling initializer.
 
         Args:
             durations: Durations of instructions to be used in scheduling.
-            dd_sequence: Sequence of gates to apply in idle spots.
+            dd_sequences: Sequence of gates to apply in idle spots.
```

Comment:
This update seems more than that explained by the PR title and not specific to IBM hardware. Do you think worth updating the terra DD rather than here?

Reply:
This DD is completely separate and does not have any dependencies on Terra. You are welcome to update Terra's DD, however, this one makes no generality guarantees. It is for IBM hardware and will evolve with it.